lwjgl3-awt pom.xml uses properties which make it unusable from gradle

lwjgl3-awt pom.xml uses now properties which make it unusable from gradle.

Steps to reproduce:

- Unzip [test.zip](https://github.com/user-attachments/files/18307219/test.zip)
- Run gradle
<pre>
$ gradle build
> Task :compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find lwjgl-3.3.5-${lwjgl.natives}.jar (org.lwjgl:lwjgl:3.3.5).
<pre>

--- a/pom.xml
+++ b/pom.xml
@@ -377,7 +377,7 @@
 			<groupId>org.lwjgl</groupId>
 			<artifactId>lwjgl-opengl</artifactId>
 			<classifier>${lwjgl.natives}</classifier>
-			<optional>true</optional>
+			<scope>test</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.lwjgl</groupId>
@@ -388,6 +388,7 @@
 			<groupId>org.lwjgl</groupId>
 			<artifactId>lwjgl</artifactId>
 			<classifier>${lwjgl.natives}</classifier>
+			<scope>test</scope>
 		</dependency>
 
 		<!-- TESTS -->

